src/Core/Components/SortableList/FluentSortableList.razor

```diff
@@ -0,0 +1,17 @@
+@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentComponentBase
+@using System.Collections.Generic
+@using System.Diagnostics.CodeAnalysis
+
+@typeparam TItem
+
+<div id="@Id" class="@ClassValue" style="@StyleValue">
+    @foreach (var item in Items)
+    {
+        @if (SortableItemTemplate is not null)
```

Comment:
Could you do the test before the `foreach`?

Reply:
If `SortableItemTemplate` is required, you could add `[EditorRequired]`

Reply:
Added EditorRequired and moved the check up